client/src/components/DetailedCard/DetailedCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import MuiDialogTitle from '@material-ui/core/DialogTitle';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+import DetailedCardForm from './DetailedCardForm';
+import { DialogContent } from './useStyles';
+import useStyles from './useStyles';
+import { Box } from '@material-ui/core';
```

Comment:
We can combine some of these imports. `import { Box, Typography, DialogTitle, . . . } from "@material-ui/core"`